pydantic/_internal/_generate_schema.py

```diff
@@ -1316,15 +1357,15 @@ def get_first_arg(type_: Any) -> Any:
         return Any
 
 
-def _get_pydantic_modify_json_schema(obj: Any) -> typing.Callable[[JsonSchemaValue], JsonSchemaValue] | None:
-    js_modify_function = getattr(obj, '__pydantic_modify_json_schema__', None)
+def _get_pydantic_modify_json_schema(obj: Any) -> GetJsonSchemaFunction | None:
```

Comment:
Could we rename this function to `_extract_get_pydantic_json_schema` or similar, I think it will help make it clearer to remove the word `modify` here, as I think that's a vestige of `__pydantic_modify_json_schema__`